add parameters 23, 29 (#31)
jooby-dev/jooby-docs: docs/parameter-types.md

Message hex dump with LRC: `03 05 12 05 00 00 c8 8c`

## Absolute data

> Parameter of that type used to setup absolute data for not multichannel device

Reviewer: ```suggestion
Parameter is used to setup absolute data for not multichannel device.
```

Author: fixed in all descriptions

### Format

| Size | Type   | Field                                          |
| ---- | ------ | ---------------------------------------------- |
| 1    | `byte` | parameter type = `23`                          |
| 1    | `byte` | [meter value](#meter-value)                    |
| 3    | `byte` | [pulse coefficient](#pulse-coefficient)        |
| 3    | `byte` | [pulse counter's value](#pulse-counters-value) |

#### **meter value**

Base absolute meter value.
<br>
It is a `4`-byte unsigned int BE.

#### **pulse coefficient**

> [Pulse coefficient](./types.md#pulse-coefficient)

Reviewer: ```suggestion
[See details](./types.md#pulse-coefficient).
```

?

Author: fixed

| interval       | `3600` | `0x100e` |

Message hex dump with LRC: `03 03 1c 10 0e 57`

## Absolute data multi channel

> Parameter of that type used to setup absolute data for multichannel device

Reviewer: ```suggestion
Parameter is used to setup absolute data for multichannel device.
```

### Format

| Size | Type   | Field                                          |
| ---- | ------ | ---------------------------------------------- |
| 1    | `byte` | parameter type = `29`                          |
| 1    | `byte` | [channel index](#channel-index)                |
| 1    | `byte` | [meter value](#meter-value)                    |
| 3    | `byte` | [pulse coefficient](#pulse-coefficient)        |
| 3    | `byte` | [pulse counter's value](#pulse-counters-value) |

> #### **channel-index**

Author: fixed

Channel index number.
<br>
It is a `1`-byte unsigned int.

#### **meter value**

Base absolute meter value.
<br>
It is a `4`-byte unsigned int BE.

#### **pulse coefficient**

> [Pulse coefficient](./types.md#pulse-coefficient)

Reviewer: as above

Author: fixed

#### **pulse counter's value**

Base pulse counter's value.
<br>
It is a `4`-byte unsigned int BE.

### Examples

> #### set absolute data (not multichannel device):

Author: fixed
